### PR TITLE
[Behavior project] Add raw Bpod output

### PR DESCRIPTION
**Pull request by weiglszonja**
- Add custom interface for converting Bpod data from Constantinople lab
- Add notes for documentation
- [Example file](https://us-east-2.console.aws.amazon.com/s3/object/catalystneuro-processing?region=us-east-2&bucketType=general&prefix=constantinople-lab-to-nwb/C005_RWTautowait_20190909_1456293.nwb)
- The processed data will be added in a follow-up PR

### CODE REVIEW

File: src/constantinople_lab_to_nwb/mah_2024/mah_2024_requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+ndx-structured-behavior @ git+https://github.com/rly/ndx-structured-behavior.git@main
```
Comment (weiglszonja) [inline src/constantinople_lab_to_nwb/mah_2024/mah_2024_requirements.txt:1]: switched to forked version until PR is merged on original
```suggestion
ndx-structured-behavior @ git+https://github.com/catalystneuro/ndx-structured-behavior.git@make_task_program_and_schema_optional
```

File: src/constantinople_lab_to_nwb/mah_2024/interfaces/mah_2024_bpodinterface.py [hunk 2]
```diff
@@ -0,0 +1,565 @@
...
+from ndx_structured_behavior.utils import loadmat
+from pynwb import NWBFile
+
+
+class Mah2024BpodInterface(BaseDataInterface):
+    """Behavior interface for mah_2024 conversion"""
+
+    def __init__(
+            self,
+            file_path: FilePathType,
```
Comment (alessandratrapani) [inline src/constantinople_lab_to_nwb/mah_2024/interfaces/mah_2024_bpodinterface.py:30]: Just a note on what came up in yesterday meeting: `FilePathType` is now deprecated in neuroconv

**Review by alessandratrapani — APPROVED** [review]
Just one small comment. Besides that, looks good! Very nice notes

File: src/constantinople_lab_to_nwb/mah_2024/interfaces/mah_2024_bpodinterface.py (hunk 2 above)
Comment (weiglszonja) [reply src/constantinople_lab_to_nwb/mah_2024/interfaces/mah_2024_bpodinterface.py:30]: Right!!

**Pull request merged**